packages/modules/itwin-viewer-react/src/services/Initializer.ts

```diff
@@ -11,17 +12,23 @@ import { UrlDiscoveryClient } from "@bentley/itwin-client";
 import { Presentation } from "@bentley/presentation-frontend";
 import { UiComponents } from "@bentley/ui-components";
 import { UiCore } from "@bentley/ui-core";
-import { AppNotificationManager, UiFramework } from "@bentley/ui-framework";
+import {
+  AppNotificationManager,
+  FrameworkReducer,
+  StateManager,
+  UiFramework,
+} from "@bentley/ui-framework";
 
 import { AppUi } from "../components/app-ui/AppUi";
 import { initRpc } from "../config/rpc";
-import store from "../store";
 import { IModelBackendOptions, ItwinViewerInitializerParams } from "../types";
 import { ai, trackEvent } from "./telemetry/TelemetryService";
 
 // initialize required iModel.js services
 class Initializer {
   private static _initialized: Promise<void>;
+  // State manager for extensions to have redux store
+  private static _appStateManager: StateManager | undefined;
```

Comment:
do we need this? It doesn't appear to be used outside of initializing a new StateManager. Can we just initialize and not store it?

Reply:
Yep, it felt weird to just initialize it without keeping it around. I don't like how the constructor sets the singleton. Removed the variable.